package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "playwright_repo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "test.js",
+  "type":"commonjs",
+  "scripts": {},
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@playwright/test": "^1.42.1",
+    "@types/node": "^20.11.28"
+  },
+  "dependencies": {
+    "playwright": "^1.42.1"
```

Comment:
I would move this to devDepts as well. No sence in having dependencies separately for this repo

Reply:
"playwright": "^1.42.1" moved to devDependencies